2017/nomadic-hues/world/world.pde

```diff
@@ -35,6 +40,8 @@ void setup() {
   colorMode(HSB, 360, 100, 100, 100);
   frameRate(60);
 
+  _fontLabel = createFont("Calluna", 48, true);
```

Comment:
hopefully it exists on the system we are running this on.

Reply:
yeah I just installed it